ja/appendices/4-0-migration-guide.rst

```diff
@@ -204,6 +202,14 @@ Http
 * ``Cake\Http\Response::getCookie()`` によって返される配列キーが変更されました。
   ``expire`` が ``expires`` に、 ``httpOnly`` が ``httponly`` に変わりました。
 
+Http\Session
```

Comment:
follow it

- Document changes regarding "SameSite" attribute for session cookies.: https://github.com/cakephp/docs/pull/6402